File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Api/Models/Feedback.cs
namespace PickleballCourtBookingSystem.Api.Models

{
    public class Feedback
    {
        public Guid? Id { get; set; }
        public float Rating { get; set; }
        public string Comment { get; set; }
        public string CustomerId { get; set; }
        public string CourtId { get; set; }
        public string BookingId { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/Feedback.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Feedback(
    var id: String,
    var rating: Float,
    var comment: String?,
    var courtId: String,
    var bookingId: String,
    var customerId: String
) : Parcelable